extending-valtimo/dashboard/custom-data-source.md

```diff
@@ -21,13 +21,13 @@ class RandomNumberWidgetDataSource {
 }
 ```
 
-The `@Component` is necessary to register the new data source as a Spring bean.
-
-The data source key `randomNumber` must be unique and is used the identify the data source. The same key must also be
+Please keep the following in mind:
+- The new data source should be registered as a Spring bean, for example with an `@Bean` or `@Component` annotation
+- The data source key `randomNumber` must be unique and is used the identify the data source. The same key must also be
```

Comment:
I think we need to clarify that `randomNumber` is the example even though it might be obvious.

```suggestion
- The data source key (in this example `randomNumber`) must be unique and is used the identify the data source. The same key must also be
```